app/src/main/java/com/android/sample/ui/fridge/FridgeScreen.kt

```diff
@@ -30,29 +117,414 @@ import com.android.sample.ui.utils.PlateSwipeScaffold
+          // Subtitle text
+          Text(
+              text = stringResource(R.string.empty_fridge_description),
+              style = MaterialTheme.typography.bodyMedium,
+              color = MaterialTheme.colorScheme.onPrimary,
+              modifier = Modifier.padding(horizontal = PADDING_32.dp).zIndex(1f),
+              textAlign = TextAlign.Center)
+
+          Spacer(modifier = Modifier.weight(2f))
+
+          // Row to hold the chef image and change its position horizontally
+          Row(modifier = Modifier.weight(1f), verticalAlignment = Alignment.CenterVertically) {
+            Spacer(modifier = Modifier.weight(1f))
+            ChefImage(modifier = Modifier.weight(18f).zIndex(-1f))
+            Spacer(modifier = Modifier.weight(1f))
+          }
+          Spacer(modifier = Modifier.weight(2f))
+        }
+
+    // Action button
+    PlateSwipeButton(
+        stringResource(R.string.add_ingredient),
+        modifier = Modifier.align(Alignment.BottomCenter),
+        onClick = {
+          userViewModel.clearIngredientList()
+          userViewModel.clearSearchingIngredientList()
+          userViewModel.clearIngredient()
+          navigationActions.navigateTo(Screen.FRIDGE_SEARCH_ITEM)
+        })
+  }
+}
+
+/**
+ * Fridge Content when not empty
+ *
+ * @param navigationActions
+ * @param paddingValues
+ * @param userViewModel
+ * @param listFridgeItem
+ */
+@Composable
+private fun FridgeContent(
+    navigationActions: NavigationActions,
+    paddingValues: PaddingValues,
+    userViewModel: UserViewModel,
+    listFridgeItem: List<Pair<FridgeItem, Ingredient>>
+) {
+  Box(modifier = Modifier.fillMaxSize().padding(paddingValues)) {
+    Column(
+        modifier = Modifier.fillMaxSize(),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally) {
+          // Row to display the title and number of items
+          Row(
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .padding(
+                          start = PADDING_32.dp,
+                          end = PADDING_32.dp,
+                          top = PADDING_16.dp,
+                          bottom = PADDING_8.dp),
+              horizontalArrangement = Arrangement.SpaceBetween) {
+                Text(
+                    text = stringResource(R.string.fridge_title),
+                    style = MaterialTheme.typography.titleMedium,
+                    fontSize = TITLE_FONT_SIZE.sp,
+                    color = MaterialTheme.colorScheme.onPrimary)
+                Text(
+                    text = "${listFridgeItem.size} items",
+                    style = MaterialTheme.typography.bodyMedium,
+                    fontSize = TITLE_FONT_SIZE.sp,
+                    color = MaterialTheme.colorScheme.onPrimary)
+              }
+
+          // Lazy Column to display the fridge items
+          LazyColumn(modifier = Modifier.fillMaxSize()) {
```

Comment:
When adding multiple ingredients to the fridge (e.g., 4), it becomes difficult to see the expiration date of the ingredient at the bottom, even when trying to scroll down (see the attached video):

https://github.com/user-attachments/assets/447d1d82-f2cb-4940-8ac4-37ee57914312

One solution could be to add a bottom padding as a modifier to the `LazyList`. The padding should match the size of the button, creating a blank space at the bottom when scrolling. This would ensure the expiration date is visible and not obstructed by the button.

Reply:
Thanks for pointing this out. I fixed it !